src/interface/swarm.cpp

```diff
@@ -533,7 +522,8 @@ void Swarm::SortParticlesByCell() {
       KOKKOS_LAMBDA(const int k, const int j, const int i) {
         int cell_idx_1d = i + nx1 * (j + nx2 * k);
         // Find starting index, first by guessing
-        int start_index = static_cast<int>((cell_idx_1d * num_active / ncells));
+        int start_index =
+            static_cast<int>((cell_idx_1d * static_cast<Real>(num_active) / ncells));
```

Comment:
Why this change? Aren't these all `int`s?

Reply:
I wanted floating point division for `num_active`/`ncells`, especially for the case when `num_active << ncells`.